Product/Production/Gateway/PatientDiscovery_10/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/_10/gateway/ws/EntityPatientDiscoveryDeferredRequestSecured.java

```diff
@@ -57,13 +57,14 @@ public EntityPatientDiscoveryDeferredRequestSecured() {
     }
 
     @OutboundMessageEvent(beforeBuilder = PRPAIN201305UV02ArgTransformer.class,
-            afterReturningBuilder = MCCIIN000002UV01EventDescriptionBuilder.class,
-            serviceType = "Patient Discovery Deferred Request", version = "1.0")
+        afterReturningBuilder = MCCIIN000002UV01EventDescriptionBuilder.class,
+        serviceType = "Patient Discovery Deferred Request", version = "1.0")
+    @Override
     public MCCIIN000002UV01 processPatientDiscoveryAsyncReq(RespondingGatewayPRPAIN201305UV02SecuredRequestType request) {
         AssertionType assertion = getAssertion(context, null);
 
         return outboundPatientDiscoveryRequest.processPatientDiscoveryAsyncReq(request.getPRPAIN201305UV02(),
-                assertion, request.getNhinTargetCommunities());
+            assertion, request.getNhinTargetCommunities(), getWebContextProperties(context));
     }
```

Comment:
Can you explain why does Entity Outbound requires WebContextProperties for PatientDiscoveryDeferredRequest service?